Classes/Utility/Accessibility.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Accessibility.swift
+//  Freetime
+//
+//  Created by Bas Thomas Broek on 22/11/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+private protocol ContentViewContaining {
+    var contentView: UIView { get }
+}
+
+extension UICollectionViewCell: ContentViewContaining {}
+extension UITableViewCell: ContentViewContaining {}
+
+enum AccessibilityHelper {
+
+    static func generatedLabel(forCell cell: UICollectionViewCell) -> String {
```

Comment:
What's the thought behind having separate methods for the cells when we could just call the `forContentViewContainer` method directly without changing parameters?

Reply:
Hmm, because I thought it could be confusing to call this and see it would expect something other than a cell; in this case the protocol. Right now it would be clear to the caller what type they can provide.